student-info-system/sis/models.py

```diff
@@ -362,36 +377,38 @@ def name_for_session(cls, abbrev):
             return sname[0]
 
     date_registration_opens = models.DateField('Registration Opens')
+    date_registration_closes = models.DateField('Registration Closes')
     date_started = models.DateField('Classes Start')
     date_last_drop = models.DateField('Last Drop')
     date_ended = models.DateField('Classes End')
 
-    semester = models.CharField('semester', choices=SESSIONS, default=FALL, max_length=6)
+    session = models.CharField('semester', choices=SESSIONS, default=FALL, max_length=6)
```

Comment:
yes, that was the point of it being about 4 commits in a single branch. All the commits are independent and single-purpose. So yes, they could have been separate branches.

But I chose to do it this way because all of them were db-breaking, and I didn't want to have to merge/remerge/remerge while waiting for them to proceed. 

The issues linked indicate the specific changes and how it impacts things.

Even now we can back that out and do it as four+ branches -- I'm OK with that.

Reply:
I wouldnt say change it this time but i think from this point forward lets not let prs get this big. Testing it becomes a hassle because it ends up taking a lot longer or we have to assign more than one person to a pr if theyre this big. If this had been smaller both jeff and I probably could have tackled it and got it done by now but since its so many changes now Its going to take me until probably sometime after work to do all the tests